compositor_render/src/renderer/scene.rs

```diff
@@ -34,7 +41,8 @@ impl RenderNode {
         match spec {
             NodeParams::WebRenderer { instance_id } => {
                 let renderer = ctx.web_renderers.get(instance_id)?;
-                Ok(Self::Web { renderer })
+                let buffers = Mutex::new(HashMap::new());
```

Comment:
We want to keep the code here minimal, consider creating WebRenderNode and moving this logic there.
Also you don't need mutex, you can change &self to &mut self in render method